### ISSUE TITLE
About eegdataset

### ISSUE THREAD

**Issue by tak0705** [alinvdu/reproduce-dream-diffusion#7]
Check the Load the .npy file
data = np.load('/content/drive/My Drive/eegdataset/eeg/10000.npy')

Where can I download the eegdataset here?

**Comment by alinvdu** [comment]
From here https://tinyurl.com/eeg-visual-classification, and preprocess it using the notebook cells to generate multiple .npy files

**Comment by alinvdu** [comment]
eeg_5_95_std.pth

**Comment by tak0705** [comment]
Thank you, I solve this problem.

In the "Use the pretrained EEG representation with existing generative checkpoints to generate images" .
An error has occurred ""ValueError: Object 'dc_ldm.models.diffusion.ddpm.LatentDiffusion' not found in the global scope."".

How did you solve it ?

**Comment by alinvdu** [comment]
Change config15.yaml and remove all the modules before LatentDiffusion and other classes. It should look something like this:

```
model:
  base_learning_rate: 1.0e-04
  target: LatentDiffusion
...
      target: UNetModel
...
    first_stage_config:
      target: AutoencoderKL
    ...
          target: torch.nn.Identity

...
      target: FrozenCLIPEmbedder

```

**Comment by tak0705** [comment]
I also changed it.
"target: torch.nn.Identity →  target: Identity"

And It error has occored.
KeyError                                  Traceback (most recent call last)
Cell In[35], line 109
    105 generative_model = eLDM(pretrain_mbm_metafile, num_voxels,
    106             device=device, pretrain_root=config.pretrain_gm_path, #logger=config.logger,
    107             ddim_steps=config.ddim_steps, global_pool=config.global_pool, use_time_cond=config.use_time_cond)
    108 # m, u = model.load_state_dict(pl_sd, strict=False)
--> 109 generative_model.model.load_state_dict(sd['model_state_dict'], strict=False)
    110 print('load ldm successfully')
    111 state = sd['state']

KeyError: 'model_state_dict'

**Comment by z520-1** [comment]
As mentioned in the paper,We have collected approximately 120,000 EEG data samples from over 400 subjects with channel ranges from 30 to 128 on theMOABB [21] platform for the EEG pre-training. 
The data sample referred to here is:eeg_5_95_std.pth.right?

**Comment by z520-1** [comment]
> eeg_5_95_std.pth

As mentioned in the paper,We have collected approximately 120,000 EEG data samples from over 400 subjects with channel ranges from 30 to 128 on theMOABB [21] platform for the EEG pre-training.
The data sample referred to here is:eeg_5_95_std.pth.right?
If so, then why is there only 12,000 pieces of data?

**Comment by Nothinkgo** [comment]
> > eeg_5_95_std.pth
> 
> As mentioned in the paper,We have collected approximately 120,000 EEG data samples from over 400 subjects with channel ranges from 30 to 128 on theMOABB [21] platform for the EEG pre-training. The data sample referred to here is:eeg_5_95_std.pth.right? If so, then why is there only 12,000 pieces of data?

Yes,I have the same question.Maybe we should find them by ourselves on MOABB.I tried,but I didn't find the answer.
